File: Backend/vietqtran.Api/vietqtran.Models/ResponseModels/PostResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using vietqtran.Models.Entities;
using vietqtran.Models.Enums;
using vietqtran.Models.ViewModels;

namespace vietqtran.Models.ResponseModels
{
	public class PostResponse
	{
		public Guid Id { get; set; }
		public Guid UserId { get; set; }
		public string Title { get; set; }
		public DateTime CreatedAt { get; set; }
		public Visibility Visibility { get; set; }
		public bool IsPinned { get; set; }
		public ICollection<string> PostImages { get; set; }
		public AppUserVM User { get; set; }
	}
}

File: ClientApp/type/responseModel/postResponse.ts
import { UserVM } from "@type/view/UserVM";
import { Visibility } from "@type/enum/Visibility";

export interface PostResponse {
	id: string;
	userId: string;
	title: string;
	createdAt: string;
	visibility: Visibility;
	isPinned: boolean;
	postImages: string[];
	user: UserVM
}